pkg/cmd/openshift-apiserver/openshiftapiserver/project.go

```diff
@@ -1,11 +1,11 @@
 package openshiftapiserver
 
 import (
+	kcoreinformers "k8s.io/client-go/informers/core/v1"
```

Comment:
I think we should start keeping this consistent with our conventions, iow. this should be `corev1informers`

Reply:
Makes sense. I have seen the `kcoreinformers` throughout the codebase, so I think it might make sense to sweep those in the future as well